helper/src/commonMain/kotlin/com/algolia/instantsearch/helper/filter/state/MutableFiltersImpl.kt

```diff
@@ -86,7 +95,7 @@ internal data class MutableFiltersImpl(
         tagGroups.filter { it.key !in groupIDs }.forEach { tagGroups.remove(it.key) }
     }
 
-    override fun toString(): String {
-        return "MutableFiltersImpl(facets=$facetGroups, tags=$tagGroups, numerics=$numericGroups)"
+    override fun add(attribute: Attribute, hierarchicalFilter: HierarchicalFilter) {
```

Comment:
What about `addOrReplace`, to convey the semantics that writing
```kotlin
add("foo", filterA)
add("foo", filterB)
```
would only leave a single filter for `"foo"`

```suggestion
    override fun addOrReplace(attribute: Attribute, hierarchicalFilter: HierarchicalFilter) {
```

Reply:
For now every method is called `add`, so I'd rather keep it that way 🤷‍♂